packages/core/src/render3/interfaces/i18n.ts

```diff
@@ -341,6 +341,11 @@ export interface TI18n {
    * DOM are required.
    */
   update: I18nUpdateOpCodes;
+
+  /**
+   * An AST representing the translated message.
```

Comment:
nit: I'd propose adding a bit more info on the purpose of the AST, so that it's clear that it's not used for anything other than hydration:
```suggestion
   * An AST representing the translated message.
   * This AST is used to reconcile i18n message (which may also contain elements)
   * with the actual DOM during hydration.
```